src/platform/mt8195/include/platform/drivers/mt_reg_base.h

```diff
@@ -0,0 +1,179 @@
+/* Copyright Statement:
+ *
+ * This software/firmware and related documentation ("MediaTek Software") are
+ * protected under relevant copyright laws. The information contained herein
+ * is confidential and proprietary to MediaTek Inc. and/or its licensors.
+ * Without the prior written permission of MediaTek inc. and/or its licensors,
+ * any reproduction, modification, use or disclosure of MediaTek Software,
+ * and information contained herein, in whole or in part, shall be strictly prohibited.
```

Comment:
IANAL but this clause (and probably others) seem really not compatible with this project.

This is of course caught by checkpatch:
```
WARNING: Missing or malformed SPDX-License-Identifier tag in line 1
```

Reply:
I miss those part, I will correct this.